controllers/ProjectController.php

```diff
@@ -52,7 +52,8 @@ public function behaviors()
                 'class' => VerbFilter::class,
                 'actions' => [
                     'publish' => ['post'],
-                    'draft' => ['post']
+                    'draft' => ['post'],
+                    'delete' => ['post'],
```

Comment:
Shouldn't `delete` action be mapped to `DELETE` HTTP method?

Reply:
Maybe you messed up with the "delete" method in the API? In app\controllers\ProjectController "delete" method should not be called through GET.